### ISSUE TITLE
Error  `Qt_5.15' not found but it is installed actually

### ISSUE THREAD

**Issue by 0xFF1E071F** [horsicq/DIE-engine#23]
I got this error while trying to run the gui version of DIEv3.00 on my linux machine:
```
./base/die: ./base/libQt5Core.so.5: version `Qt_5.15' not found (required by /usr/lib/libQt5OpenGL.so.5)
```
I have latest Qt5 libraries and ncurses libraries are installed

**Comment by horsicq** [comment]
Thanks a lot for the bugreport! I ll try to fix it ASAP.

**Comment by 0xFF1E071F** [comment]
> Thanks a lot for the bugreport! I ll try to fix it ASAP.

Thank you so much for this great tool =)

**Comment by 0xFF1E071F** [comment]
> I fixed this issue: [20a4a44](https://github.com/horsicq/DIE-engine/commit/20a4a440eb80b7e9bc41959e0aef11d3b98c45f7)
> Could you try this file? https://github.com/horsicq/DIE-engine/releases/download/3.00/die_lin64_portable_3.00.tar.gz

Perfect! Working with no errors right now.
Thank you.
Now closing the issue =)
